cabal.project

```diff
@@ -63,3 +63,7 @@ package plutus-scripts-bench
 -- temporary! Please read the section in CONTRIBUTING about updating dependencies.
 
 -- `smtp-mail` should depend on `crypton-connection` rather than `connection`!
+
+allow-newer:
```

Comment:
Where does this dependency come from?

Reply:
Its for the tracers stuff, but its no longer needed.